scene/main/node.h

```diff
@@ -213,8 +213,8 @@ class Node : public Object {
 
 	Ref<MultiplayerAPI> multiplayer;
 
-	void _print_tree_pretty(const String &prefix, const bool last);
-	void _print_tree(const Node *p_node);
+	String _get_tree_string_pretty(const String &prefix, const bool last);
```

Comment:
```suggestion
	String _get_tree_string_pretty(const String &p_prefix, bool p_last);
```
Since the method is replaced here let's apply prefixing arguments with `p_` and no `const` for simple types